### ISSUE TITLE
Justin.tv Bug - Invalid Embed code

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bonesoul/blizztv#150]
```
When attempting to watch a Jtv stream in the Internal Player on Win Xp on 
v0.18b, you receive an Invalid Embed Code error.

Screenshot: http://i.imgur.com/OwcpS.jpg
```

Original issue reported on code.google.com by `kb39...@yahoo.com` on 21 Jan 2011 at 9:41

**Comment by GoogleCodeExporter** [comment]
```
We're aware of the bug that only happens with some computers, trying to get a 
solution from justin.tv support; 
http://community.justin.tv/forums/showthread.php?t=5604.
```

Original comment by `shalafir...@gmail.com` on 22 Jan 2011 at 10:05
- Changed state: **BugReport**
- Added labels: **Module-Streams**, **Milestone-0.20-Stable**, **Usability**

**Comment by GoogleCodeExporter** [comment]
Original comment by `shalafir...@gmail.com` on 22 Jan 2011 at 10:05

**Comment by GoogleCodeExporter** [comment]
```
Fixed as of 0.20-beta.
```

Original comment by `shalafir...@gmail.com` on 11 Feb 2011 at 12:30
- Changed state: **NeedsTesting**
- Added labels: **Milestone-0.20-Beta**
- Removed labels: **Milestone-0.20-Stable**
